AnsibleTestCRD.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: test.openstack.org/v1beta1
+kind: AnsibleTest
+metadata:
+  name: rhel-ai-validation-ansibletest
+  namespace: openstack
+spec:
+  ansibleGitRepo: https://github.com/openstack-k8s-operators/rhelai-validation.git
+  # yamllint disable rule:line-length
+  ansibleInventory: |
+    [control_node]
+    localhost ansible_connection=local
+    [target_nodes]
+    rhelai0 ansible_host=192.168.122.222 ansible_user=cloud-user ansible_ssh_private_key_file=~/rhel-ai.pem ansible_ssh_common_args='-o StrictHostKeyChecking=no'
+  # yamllint enable rule:line-length
+  ansiblePlaybookPath: main.yaml
+  ansibleVarFiles: |
+    deploy_rhelai_private_key_file: ~/rhel-ai.pem
+    model_tests_enabled: false
```

Comment:
Model tests are disabled at the moment because I haven't experimented with passing in the creds.yaml - I'm targeting junitXML at the moment and we can return to this later.